Monika After Story/game/script-topics.rpy

```diff
@@ -14435,3 +14435,73 @@ label monika_language_nuances:
     m 1tsu "So good day to you, [player].{w=0.3} {nw}"
     extend 1hub "Ahaha~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_aliens",
+            category=["misc", "society"],
+            prompt="Do you believe in aliens?",
+            pool=True
+        )
+    )
+
+label monika_aliens:
+
+    m 2eud "Do I believe in the existence of Aliens?"
+    m 4lksdla "That's kind of a random thing to ask, [player]."
+    m 4eua "...Though I can see why you would be interested to know about my opinion on the matter."
+    m 4eub "This is one of the greatest mysteries of all time, isn't it? Whether we are truly alone in this universe or not, I mean."
+    m 1rksdlc "...I know I used to wonder about that myself a lot, before you came along."
+
+    if mas_isMoniEnamored(higher=True):
+        m 1ekb "But that's in the past now. And I could never thank you enough for that."
+
+    elif mas_isMoniNormal(higher=True):
+        m 1eka "Though I don't really have to anymore, thanks to you."
+
+    elif mas_isMoniUpset(lower=True):
+        m 1dkc "I just hope I won't get to feel that way ever again."
+        
+    elif mas_isMonibroken():
+        m 1rkc "..."
+
+    m 3euc "Anyway, what I meant by that is, we've all probably asked ourselves what's up there at least once, right?"
+    m 3dua "Looking at the stars always fills you with a sense of wonder and mystery. It's really no surprise that so many people are passionate about this subject."
+    m 2eua "But to answer your question..."
+    m 2eud "I do believe, or at least want to believe, that there has to be {i}something{/i} out there."
+    m 2rksdla "I guess part of it has to do with me finding the idea of us being the only ones rather depressing, {w=0.3}{nw}"
+    extend 2eud "but when you think about it a little, it doesn't sound so unlikely..."
+    m 4eksdla "After all, saying the universe is vast would be {i}quite{/i} the understatement."
+    m 4eua "All you need is just one planet with the right conditions and environment to be favorable for life to develop, right?"
+    m 4esb "And there are 8 planets in our system alone... {w=0.5}{nw}" 
+    extend 4eub "...But of course, there are many more star systems, each with their own planets within them."
+    m 4dsc "Now, consider that our galaxy, the 'Milky Way,' supposedly contains hundreds of billions of stars. And that's only counting one galaxy, obviously..."
```

Comment:
```suggestion
    m 4dsc "Now, consider that our galaxy, the 'Milky Way,' contains hundreds of billions of stars. And that's only counting one galaxy, obviously..."
```
I wouldn't really use 'supposedly' here. Obviously there're differences between various estimates of that number, but the order of magnitude is pretty consistent.